react/test/exampleTest.js

```diff
@@ -0,0 +1,5 @@
+describe('example test', () => {
+  it('should pass', () => {
+    expect(true).toBe(true);
+  });
+});
```

Comment:
This is fine for checking the set up, but since this PR contains `FormContainer`, you'll need to test that it is working properly.